### DISCUSSION TITLE
Fallback URL in formula?

### DISCUSSION THREAD

**Discussion by dpo** [Homebrew/discussions#3561] [Writing Formulae/Casks] [answered]
This summer, I've found that several servers for sources I use in my taps have been down. I have a vague recollection that it used to be possible to specify a mirror URL in a Homebrew formula a long time ago (I may be wrong though). Is that still possible? What would the syntax be?

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
```
mirror "url"
```

**Comment by carlocab** [comment]
Try `mirror`
```ruby
  url "https://curl.se/download/curl-7.84.0.tar.bz2"
  mirror "https://github.com/curl/curl/releases/download/curl-7_84_0/curl-7.84.0.tar.bz2"
  mirror "http://fresh-center.net/linux/www/curl-7.84.0.tar.bz2"
  mirror "http://fresh-center.net/linux/www/legacy/curl-7.84.0.tar.bz2"
```

**Comment by dpo** [comment]
Thank you. I was looking for that. For some reason, couldn't find it in the docs. I must be blind.
